README.md

```diff
@@ -1,6 +1,6 @@
 # SQL style guide
 
-**[Read the guide](http://www.sqlstyle.guide)**
+**[Read the guide](includes/sqlstyle.guide.md)**
 
 ---
```

Comment:
yeah i noticed that; wasn't sure if that would be a branch only error.  

also what link do you mean exactly re: the 2nd comment?